src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/components/_component.scss

```diff
@@ -42,8 +11,8 @@
     border-top: solid 1px $color-border;
     margin-bottom: 0;
     line-height: 30px;
-    padding-top: 8px;
-    padding-bottom: 8px;
+    padding-top: 3px;
```

Comment:
should we create an super tiny padding? seems like we use 3 a lot, you could even keep it confined to this component.

Reply:
done.